whisper.py

```diff
@@ -95,6 +95,9 @@ def _py_fallocate(fd, offset, len_):
 CACHE_HEADERS = False
 AUTOFLUSH = False
 FADVISE_RANDOM = False
+# Buffering setting applied to all operations that do *not* require
+# a full scan of the file in order to minimize cache thrashing.
+BUFFERING = 0
```

Comment:
After thinking about it, 0 might be more like "performance related bugfix" than "weird tunable". If it's *always* the case that we read extra data, and we *never* need that data, then there's no point to have read that extra data. That's not something you'd tune.

That said, I'd like to see some before/after data (graphs!) showing the effect of this change to confirm my assumptions.